.github/workflows/master_build.yml

```diff
@@ -26,6 +26,10 @@ jobs:
         pip install -U Sphinx==3.5.4
         pip install --upgrade myst-parser
         pip install sphinx_rtd_theme
+        pip install 'jinja2<3.1'
+        pip install 'myst-parser==0.16.0'
```

Comment:
Why precisely those version numbers?

Reply:
It is no longer supported due to the jinja2 specification.
See below for source code diffs.

`$ diff jinja_3_1_0/src/jinja2/__init__.py jinja_3_0_3/src/jinja2/__init__.py
16a17,19
> from .filters import contextfilter
> from .filters import environmentfilter
> from .filters import evalcontextfilter
30a34,37
> from .utils import contextfunction
> from .utils import environmentfunction
> from .utils import escape
> from .utils import evalcontextfunction
31a39
> from .utils import Markup
37c45
< __version__ = "3.1.0"
---
> __version__ = "3.0.3"`


There is also an announcement on the official jinja2 Github page .
Please refer to that as well.

announcement on the official jinja2 github page here :
https://github.com/pallets/jinja/issues/1630

Reply:
It is no longer supported due to the jinja2 specification.
See below for source code diffs.

$ diff jinja_3_1_0/src/jinja2/__init__.py jinja_3_0_3/src/jinja2/__init__.py  -u
--- jinja_3_1_0/src/jinja2/__init__.py  2022-05-28 15:55:09.596198699 +0900
+++ jinja_3_0_3/src/jinja2/__init__.py  2022-05-28 15:55:57.171974703 +0900
@@ -14,6 +14,9 @@
 from .exceptions import TemplatesNotFound as TemplatesNotFound
 from .exceptions import TemplateSyntaxError as TemplateSyntaxError
 from .exceptions import UndefinedError as UndefinedError
+from .filters import contextfilter
+from .filters import environmentfilter
+from .filters import evalcontextfilter
 from .loaders import BaseLoader as BaseLoader
 from .loaders import ChoiceLoader as ChoiceLoader
 from .loaders import DictLoader as DictLoader
@@ -28,10 +31,15 @@
 from .runtime import StrictUndefined as StrictUndefined
 from .runtime import Undefined as Undefined
 from .utils import clear_caches as clear_caches
+from .utils import contextfunction
+from .utils import environmentfunction
+from .utils import escape
+from .utils import evalcontextfunction
 from .utils import is_undefined as is_undefined
+from .utils import Markup
 from .utils import pass_context as pass_context
 from .utils import pass_environment as pass_environment
 from .utils import pass_eval_context as pass_eval_context
 from .utils import select_autoescape as select_autoescape

-__version__ = "3.1.0"
+__version__ = "3.0.3"


There is also an announcement on the official jinja2 Github page .
Please refer to that as well.

the announcement on the official jinja2 Github page here :
https://github.com/pallets/jinja/issues/1630